keyboards/xdboards/recon/keyboard.json

```diff
@@ -0,0 +1,209 @@
+{
+    "manufacturer": "XD Boards",
+    "keyboard_name": "xdboards/recon",
+    "maintainer": "Xanimos",
+    "development_board": "promicro",
+    "pin_compatible": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "oled": true
+    },
+    "matrix_pins": {
+        "cols": ["D4", "C6", "D7", "E6", "B4", "B5"],
+        "rows": ["F4", "F5", "F6", "F7", "B1"]
+    },
+    "url": "https://xdboards.xyz",
+    "tags": ["columnar", "split", "thumb cluster", "rgb", "encoder"],
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x02F7",
+        "vid": "0xFEED"
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "B6","pin_b": "B2" }
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "serial": {
+            "pin": "D3"
+        },
+        "handedness": {
+            "pin": "D2"
+        },
+        "matrix_pins": {
+            "right": {
+                "cols": ["D4", "C6", "D7", "E6", "B4", "B5"],
+                "rows": ["F4", "F5", "F6", "F7", "B1"]
+            }
+        },
```

Comment:
```suggestion
```

Reply:
Nifty! Didn't put together it was optional if they were the same based on the [info.json reference doc](https://docs.qmk.fm/reference_info_json#split-keyboard) However I found the doc for it being optional in the [Config Options](https://docs.qmk.fm/config_options#other-options)

Reply:
@zvecr Thank you for your speedy review! I cleaned up the documents with your requested changes when you have a chance to take another look.